packages/header-checker-lint/README.md

```diff
@@ -1,9 +1,28 @@
 # header-checker-lint
 
-> A GitHub App built with [Probot](https://github.com/probot/probot) that ensures source files
-contain a valid license header.
+> A GitHub App built with [Probot][probot] that ensures source files contain a
+valid license header.
 
-## Setup
+## Installation
+
+You can install this bot [from GitHub][github-app-link].
+
+### Configuration
+
+To configure the bot, you can create a configuration file:
+`.bots/header-checker-lint.json`. The contents of this file define a JSON object
```

Comment:
clever, I like the `.bots/` folder, and then each config just maps to a bot?

Reply:
Yeah, this layout is totally up for discussion, though.